lib/sqlite3/sqlite3.nit

```diff
@@ -305,8 +305,9 @@ redef universal Int super Sqlite3Data end
 
 redef universal Float super Sqlite3Data end
 
-redef class String
-	super Sqlite3Data
+redef class String super Sqlite3Data end
```

Comment:
I wondered the same thing, but `Sqlite3Data` is used only as a return type to `StatementEntry:::value`, it has no services. So this tells you that the method can return a `String`. Dynamically you can still check it as being a `Text` if you want.

This was more of a cheap way to document the return type of the method. Today I consider it more of an abuse of multiple inheritance and class refinement. It would have been as efficient and as clear to return a `nullable Object` and document the possible dynamic types.